docker.d/init_worker.sh

```diff
@@ -9,6 +27,8 @@ case $COT_PRODUCT in
         test $GOOGLE_CREDENTIALS_FIREFOX_DEP
         export GOOGLE_CREDENTIALS_FIREFOX_DEP_PATH=$CONFIG_DIR/dep.p12
         echo $GOOGLE_CREDENTIALS_FIREFOX_DEP | base64 -d > $GOOGLE_CREDENTIALS_FIREFOX_DEP_PATH
+
+        import_cert dep $CERT_DIR/dep.pem
         ;;
       prod)
         test $GOOGLE_PLAY_SERVICE_ACCOUNT_FIREFOX_RELEASE
```

Comment:
Where are these environment variables defined?

Reply:
In GCP/Kubernetes